node.js/app.js

```diff
@@ -32,33 +32,41 @@ async function main() {
     const network = await gateway.getNetwork(kclient.channel.name);
     const contract = network.getContract(chaincodeName);
 
-    const isInit = prompt('Calling "InitLedger" (y/n)? ');
-
-    if (isInit === 'y') {
-      // Initialize ledger with demo assets
-      const fcn = 'InitLedger';
-      console.log(`--> Submitting Transaction. fcn: ${fcn}`);
-      await contract.submitTransaction(fcn);
-      console.log('*** Result: committed');
+    let isInit;
+    if (process.env.IS_INIT) {
+      isInit = process.env.IS_INIT;
     } else {
-      // Add an asset with a random id to the channel using the 'CreateAsset' chaincode function
-      let fcn = 'CreateAsset';
-      const assetId = `asset-${Math.floor(Math.random() * 1000000)}`;
-      console.log(`Adding new asset with following random asset ID to the ledger: ${assetId}`);
-      let args = [assetId, "yellow", "5", "Benny", "53000"];
-
-      console.log(`--> Submitting Transaction. fcn: ${fcn}, args: ${args}`);
+      isInit = prompt('Calling "InitLedger" (y/n)? ');
+    }
+    let NUM_ITERATIONS = process.env.NUM_ITERATIONS ? parseInt(process.env.NUM_ITERATIONS) : 1000;
+    for (let i = 0; i < NUM_ITERATIONS; i++) {
+      let fcn, args, assetId;
+      if (isInit === 'y') {
```

Comment:
this should check `i == 0` so that `InitLedger` is only called once for the batch of iterations.